### PR TITLE
Some visual changes

### PR DESCRIPTION
**Pull request by sahej172002**

### CODE REVIEW

File: challenge/src/mandalorian/BabyYoda.css [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 .baby-yoda h1 {
     font-family: Impact, sans-serif;
+    font-size: 72px;
```
Comment (mjrb) [inline challenge/src/mandalorian/BabyYoda.css:19]: this messed up the alignment on the baby yoda component. now the image is still left aligned but the text is longer so it looks a little wonky. maybe try going into babyyoda.jsx and see if you can center that image

**Review by mjrb — CHANGES_REQUESTED** [review]
i looked at the diff and saw flaming text, i immediately fell in love with this pr.
however, it broke the baby yoda component a little, and it looks a bit wonky.
see coments for suggestions

**Review by sahej172002 — COMMENTED** [review]
Made the changes in the latest commit

File: challenge/src/mandalorian/BabyYoda.css (hunk 1 above)
Comment (sahej172002) [reply challenge/src/mandalorian/BabyYoda.css:19]: Made the changes in the latest commit

**Comment by mjrb** [comment]
i think you pretty much had the right idea with the margin: auto stuff, but it still looked a bit wonky on my screen. let me know how it looks on your screen and then let me know @sahej172002

**Review by sahej172002 — COMMENTED** [review]
I checked it on my screen and it seems to work better when the margin is in the containing div instead

**Review by sahej172002 — COMMENTED** [review]
Changes are ok but I cant seem to push any commits from my command line

**Comment by mjrb** [comment]
> Changes are ok but I cant seem to push any commits from my command line

what error do you get from the command line?

File: challenge/public/index.html [hunk 2]
```diff
@@ -5,23 +5,32 @@
...
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <link rel="stylesheet" type="text/css" href="%PUBLIC_URL%/styles.css"/>
     <title>HackRU Reactment Sandbox</title>
-    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Comfortaa:bold&display=swap&effect=fire-animation">
     <style>
-        body {
-            font-family: 'Montserrat', sans-serif;
-        }
+      body {
```
Comment (mjrb) [inline challenge/public/index.html:16]: potentially after you created this pull this project got a main css file under `challenge/public/styles.css`, these styles should be moved in there

**Review by mjrb — CHANGES_REQUESTED** [review]
sorry, i know i said this looked fine but i actually saw some changes i wanted.
they should be fairly quick tho

File: challenge/public/index.html (hunk 2 above)
Comment (sahej172002) [reply challenge/public/index.html:16]: Oh I'll try to figure that then

**Comment by shivanmodha** [comment]
Closing this

**Pull request closed without merging**